www/stos02/ukol_7/main.css

```diff
@@ -0,0 +1,42 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: sans-serif;
+    text-align: center;
+}
+
+hr {
+    border-style: solid;
+    border-color: #eee;
+}
+
+#game-field {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 1024px;
+    margin: 0 auto;
+}
+
+.card {
+    background-color: #41cc9e;
+    color: transparent;
+    border: 15px solid white;
+    width: 20%;
+    height: 150px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    border-radius: 25px;
```

Comment:
kdyz pouzijes vlastnost `user-select: none` tak Ctrl+A uz prestane delat problemy

Reply:
pokud bych přidala pouze do css  user-select: none;  bez podmínky v js, tak mi to pořád dělá problém.

Reply:
jakej problem? to ze hrac stale muze podvadet?

Reply:
jo, že se mu tam přičítají body, když kliká na stejnou kartu

Reply:
tak v tom pripade proste pridas podminku na zacatku kliknuti, ze pokud je karta jiz otocena tak hned ukoncis funkci

Reply:
jo to bych měla mít v tom opraveným. Ještě teda přidám user-select: none; Děkuju